pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1421,13 +1431,16 @@ func PublishBlobInfoToZedCloud(ctx *zedagentContext, blobSha string, blobStatus
 	//We queue the message and then get the highest priority message to send.
 	//If there are no failures and defers we'll send this message,
 	//but if there is a queue we'll retry sending the highest priority message.
-	zedcloud.SetDeferred(zedcloudCtx, blobSha, buf, size, statusURL,
+	const ignoreErr = LOCDest
+	ditems := createDeferredItemsByDest(ctx, dest, ignoreErr, blobSha, buf, size,
 		true, false, info.ZInfoTypes_ZiBlobList)
-	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, true)
+	zedcloud.AddDeferredList(zedcloudCtx, ditems)
+	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, false)
```

Comment:
Why are you changing the last argument to HandleDeferred from true to false here?